auth-connector/src/main/java/com/sequenceiq/cloudbreak/auth/altus/GrpcUmsClient.java

```diff
@@ -310,6 +309,14 @@ public boolean checkRight(String actorCrn, String userCrn, String right, String
             LOGGER.info("Letting read operation through for right {} for user {}!", right, userCrn);
             return true;
         }
+        return false;
+    }
+
+    @Cacheable(cacheNames = "umsUserRightsCache", key = "{ #actorCrn, #userCrn, #right, #resource }")
```

Comment:
let's not cache anything on the client. It complicates reasoning about the state of the world and if caching is needed is should be done in the UMS. The actual network hoop is not that expensive I don't think.

Reply:
This is cached in the existing code and I didn't want to change it. I did dig through the cache initialization and it is set with an expiration of 1s. We can open a JIRA to remove the cache separately from this JIRA.

Reply:
We definitely need caching, we had serious performance issues if we do not used caching, it has small expiration time so I do not see what could be the problem with that.